### ISSUE TITLE
ImportError: cannot import name 'jab_to_rg'

### ISSUE THREAD

**Issue by am-ac** [ksmet1977/luxpy#14]
Not sure if that's and issue with my dependencies or the luxpy package itself. However, I get above error with version > 1.6 (1.6.0 and 1.6.1) Versions up to 1.5.3 work fine. Installed in Thonny with it's own packaging system.

**Comment by ksmet1977** [comment]
From v1.6.0 onwards there have been some changes in the way the cri helper functions work (some don't exist). This was implemented to hopefully make luxpy quite a bit faster in many cases and to make development of further functionality more easy. I made sure the basic functionality like spd_to_cri, spd_to_DEi, spd_to_rg remained, but for calls to some of the other functions the changes would be breaking (hence move from version 1.5.x to 1.6.x) as these no longer exist. 
I'm assuming you're calling/importing  jab_to_rg yourself?
The way the cri subpackage works is that all functions requiring some form of hue-bin slicing call a function _get_hue_bin_data(jabt, jabr,.....). This function has pretty much the same functionality as the former gamut_slicer() function, but avoids some loops to maker it faster in most cases. The output of _get_hue_bin_data() is a dictionary with all sorts of data related to the hue bins. That dictionary should then be used as input to _hue_bin_data_to_rg() to get for example Rg, or to _hue_bin_data_to_rxhj() to get the local color fidelity indices, local hue shifts and local chroma shifts., ...
To get Rg there's a dedicated spd_to_rg() function, but the easiest and fastest way (if you also want Rf or other metric values) is to call spd_to_cri(spd, out ='Rf,Rg') as this will avoid double calls to the calculation of _get_hue_bin_data().
In principle the function you want would be something like:
```
def jab_to_rg(jabt, jabr, start_hue = 0, nhbins = 16, max_scale = 100, normalized_chroma_ref = 100, normalize_gamut = False):
    hue_bin_data = _get_hue_bin_data(jabt, jabr,\
                                    start_hue = start_hue, \
                                    nhbins = nhbins, \
                                    normalized_chroma_ref = normalized_chroma_ref)
  return _hue_bin_data_to_rg(hue_bin_data, max_scale = max_scale, normalize_gamut = normalize_gamut)
```
I could restore functionality for jab_to_rg() and jab_to_rhi() in a next version, but that would invite development of slower functions as now the hue bin calculation would be done twice. In addition, some keyword arguments (eg. ordered_and_sliced) would not be available as these are not required anymore with the new _get_hue_bin_data() function.

I hope this helps and if you have any questions don't hesitate to ask.

**Comment by am-ac** [comment]
Thanks for your quick reply.
I am not calling the function myself. It basicylly happens when importing luxpy. I just created a fresh ubuntu 20.04 VM and only installed thonny yet. Added luxpy via thonnys own package manager. My Script consists of one line
`
import luxpy
`
That's my error:
`
%Run TestingLuxpy.py

Traceback (most recent call last):
  File "/home/andre/playground/TestingLuxpy.py", line 1, in <module>
    import luxpy
  File "/home/andre/.local/lib/python3.8/site-packages/luxpy/__init__.py", line 240, in <module>
    from luxpy.color.cri import colorrendition as cri
  File "/home/andre/.local/lib/python3.8/site-packages/luxpy/color/cri/__init__.py", line 209, in <module>
    from .colorrendition import *
  File "/home/andre/.local/lib/python3.8/site-packages/luxpy/color/cri/colorrendition.py", line 215, in <module>
    from .indices.indices import *
  File "/home/andre/.local/lib/python3.8/site-packages/luxpy/color/cri/indices/indices.py", line 158, in <module>
    from .mcri import _MCRI_DEFAULTS, spd_to_mcri
  File "/home/andre/.local/lib/python3.8/site-packages/luxpy/color/cri/indices/mcri.py", line 30, in <module>
    from ..utils.helpers import jab_to_rg
ImportError: cannot import name 'jab_to_rg' from 'luxpy.color.cri.utils.helpers' (/home/andre/.local/lib/python3.8/site-packages/luxpy/color/cri/utils/helpers.py)
`

**Comment by ksmet1977** [comment]
That's strange as line 30 of mcri.py is part of the old version (so pre v.1.6.0). I checked the code and in the new versions this line has been changed to import `_get_hue_bin_data` (although it should also have imported `_hue_bin_data_to rg`, so I added that and updated the code on github, not yet on pypi). The updated version on github can be installed using pip as:
``` pip install git+https://www.github.com/ksmet1977/luxpy -U ```
I don't know about thonny, maybe it is some cached files, but it would be strange that it mixes versions. So I've just downloaded thonny and tried myself. I'm indeed also getting the same error. I'll try pushing a new version to pypi and see what happens.
